heat/files/mitaka/heat.conf.Debian

```diff
@@ -445,8 +445,8 @@ auth_url = http://{{ server.identity.host }}:35357
 username = {{ server.identity.user }}
 password = {{ server.identity.password }}
 user_domain_name = default
-project_domain_id=Default
-user_domain_id=Default
+project_domain_id = {{ controller.identity.get('domain', 'default') }}
```

Comment:
That's fine, but see first line in that jinja template (import from `map.jinja`), it's importing a server, not controller (as in case of heat there's only server role). So you need to change that accordingly (`server.identity.get...`) instead of `controller` otherwise it will fail.